lib/helpers.js

```diff
@@ -101,3 +99,43 @@ export function getMultibaseMultikeyHeader({value}) {
   }
   return value.slice(0, 4);
 }
+
+export function addX25519Context({contexts, keyAgreementKeyPair}) {
+  const {type} = keyAgreementKeyPair;
+  switch(type) {
+    case 'X25519KeyAgreementKey2019': {
+      if(!contexts.includes(X25519KeyAgreementKey2019.SUITE_CONTEXT)) {
+        contexts.push(X25519KeyAgreementKey2019.SUITE_CONTEXT);
+      }
+      break;
+    }
+    case 'X25519KeyAgreementKey2020': {
```

Comment:
This is fine for now -- we'll want to enable external callers to provide this information in the future so we don't have algorithm-specific deps in this library.